scripts/cita.sh

```diff
@@ -32,7 +32,7 @@ SERVICES=( forever auth bft chain executor jsonrpc network )
 SCRIPT=$(basename "$0")
 COMMAND=$1
 NODE_NAME=$2
-NODE_PATH="$(dirname "$CITA_BIN")/${NODE_NAME}"
+NODE_PATH="$(realpath ${NODE_NAME})"
```

Comment:
```suggestion
NODE_PATH=$(realpath "${NODE_NAME}")
```

And check the https://github.com/cryptape/cita/blob/develop/scripts/cita.sh#L6-L10

Reply:
updated

Reply:
https://unix.stackexchange.com/questions/136494/whats-the-difference-between-realpath-and-readlink-f